pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
```

Comment:
I think leaving open the possibility for each member pod to have it's own unique client certificate would be useful. In that case, the simplest scheme is that every node gets a client key/cert and a peer key/cert. The most common case would probably be that all the client-interface key/certificate are identical, but I do believe leaving the option open for them to be distinct is a worthwhile design goal.